Backend/ManagerLayer/Login/UserLoginManager.cs

```diff
@@ -0,0 +1,97 @@
+﻿using DataAccessLayer.Database;
+using DataAccessLayer.Models;
+using ManagerLayer.AccessControl;
+using ManagerLayer.Models;
+using ManagerLayer.UserManagement;
+using ServiceLayer.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ManagerLayer.Login
+{
+    public class UserLoginManager
+    {
+        IPasswordService _passwordService;
+        UserManagementManager _userManagementManager;
+        AuthorizationManager _authorizationManager;
+
+        public LoginManagerResponsePOCO Login(string username, string password)
+        {
+            LoginManagerResponsePOCO response;
+            _userManagementManager = new UserManagementManager();
+            _passwordService = new PasswordService();
+            var user = _userManagementManager.GetUser(username);
+            if (user == null)
+            {
+                response = new LoginManagerResponsePOCO
+                {
+                    Data = null,
+                    Message = "User does not exist",
+                    Timestamp = DateTime.UtcNow
+                };
+                return response;
+            }
+            var StoredPassword = user.PasswordHash;
+            var AttemptedPassword = _passwordService.HashPassword(password, user.PasswordSalt);
+            if (StoredPassword == AttemptedPassword)

```

Comment:
You will never be handling the password. This logic should be removed now that authorization is handled by the SSO only.

Reply:
This was meant for individual app login but can be removed due to change in requirements